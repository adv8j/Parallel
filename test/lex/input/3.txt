@TaskGroup g1
{
    @TaskGroup g2
    {
        @ Task t1
        {
            _task;
        }
    }
}
