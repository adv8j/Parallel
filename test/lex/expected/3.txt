TASKGROUP
IDENTIFIER
LBRACE
TASKGROUP
IDENTIFIER
LBRACE
Syntax Error!!!
IDENTIFIER
IDENTIFIER
LBRACE
SHARED_IDENTIFIER
SEMICOLON
RBRACE
RBRACE
RBRACE
